opent2t.p.hub/com.wink.hub/js/thingTranslator.js

```diff
@@ -361,7 +369,6 @@ class Translator {
             .catch(function (err) {
                 console.log("Request failed to: " + options.method + " - " + options.url);
                 console.log("Error            : " + err.statusCode + " - " + err.response.statusMessage);
-                // todo auto refresh in specific cases, issue 74
```

Comment:
@arjun-msft , I couldn't find issue 74 - where is this referenced; Since the 'auto' part of refresh would be within the cloud federated service, I guess the comment doesn't quiet apply? What might still apply and what I am not quite sure yet is if any specific error code is returned by WINK that we need to make the caller (Federated service in this case) Aware of to allow them to call into the refreshAuthToken() API.